null_safety_examples/cookbook/lists/mixed_list/lib/main.dart

```diff
@@ -67,14 +67,16 @@ class HeadingItem implements ListItem {
 
   HeadingItem(this.heading);
 
+  @override
   Widget buildTitle(BuildContext context) {
     return Text(
       heading,
       style: Theme.of(context).textTheme.headline5,
     );
   }
 
-  Widget buildSubtitle(BuildContext context) => null;
```

Comment:
I have no idea how this was working before. CI should have detected it :thinking: